triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/ConfigSpec.scala

```diff
@@ -20,8 +20,6 @@ import scala.language.implicitConversions
 
 class ConfigSpec extends AsyncWordSpec with Matchers with CantonFixture {
 
-  final override protected lazy val devMode: Boolean = true
```

Comment:
:+1:

Reply:
Can someone explain this for me please.

Reply:
Setting `devMode` to be true means Canton logging will be kept.

Reply:
Ah, not quite correct (it used to be mind!). `devMode` now means our patched version of Canton is used. Previously (IIRC?), the same setting had been used where `cantonFixtureDebugMode` is now used.

Reply:
> devMode now means our patched version of Canton is used.

@carlpulley-da you are confusing with `cantonFixtureDebugMode`

`devMode = true` *always* meant our patched version of Canton is used (+ canton is started in `dev` mode). 
Such tests that use the patched version of Canton need be mark with the tag `dev-canton-test` in the bazel Build file which was not the case of this ConfigTest. 

Taking into account ConfigTest test does not seems to used any `dev` feature, it seems more natural to remove the `devMode = true` that to mark it with `dev-canton-test`